build.js

```diff
@@ -0,0 +1,21 @@
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+
+rollup.rollup({
+    entry: './index.js',
```

Comment:
You should define jquery as an external dep here (see also [backbone-http](https://bitbucket.org/codeyellow/backbone-http/src/eb09b38c50aeba3f68bd3f41cb82ffd73d5b0db8/build.js?fileviewer=file-view-default))